BookingsApi/BookingsApi.IntegrationTests/Helper/TestDataManager.cs

```diff
@@ -281,6 +281,25 @@ public async Task ClearSeededHearings()
             }
             _seededHearings.Clear();
         }
+        
+        public async Task ClearJudiciaryPersonsAsync(IEnumerable<Guid> ids)
+        {
+            foreach (var externalRefId in ids)
+            {
+                try
+                {
```

Comment:
you could use Linq and Contains to remove everyone who exists with the matching ID like below to avoid the need to catch exceptions when a person does not exist. 
`
await using var db = new BookingsDbContext(_dbContextOptions);
var existing = db.JudiciaryPersons.Where(x => ids.Contains(x.Id));
db.JudiciaryPersons.RemoveRange(existing);
await db.SaveChangesAsync();`

Reply:
Need to print out when a externalRefId is not existing in the DB, like the other methods in the class